ipaserver/install/installutils.py

```diff
@@ -744,10 +749,9 @@ def read_replica_info(dir_path, rconfig):
 
     rconfig is a ReplicaConfig object
     """
-    filename = dir_path + "/realm_info"
-    fd = open(filename)
+    filename = os.path.join(dir_path, "/realm_info")
```

Comment:
One shall not `os.path.join()` with `/` (causes `ipa-replica-install` on DL0 to fail)